learning/step_by_step/your_first_game.rst

```diff
@@ -750,6 +759,10 @@ sync with the changing score:
 Now you're ready to play! Click the "Play the Project" button. You will
 be asked to select a main scene, so choose ``Main.tscn``.
 
+.. warning:: If you get any errors in the debugger saying "Node not found"
+             check the script to see if it has the `exact` same name as the node in the scene.
+             (``$`` and ``get_node()`` are `case sensitive!`)
+
```

Comment:
I don't think it's necessary to repeat the same warning twice. See above.

Reply:
Okay. Should I place the warning here or above?

Reply:
Above makes the most sense. It's where they're likeliest to run into the issue.

Reply:
Okay. I'll remove this one then.